assets/js/atomic/blocks/product-elements/title/block.tsx

```diff
@@ -120,7 +124,8 @@ export const Block = ( props: Props ): JSX.Element => {
 				disabled={ ! showProductLink }
 				name={ product.name }
 				permalink={ product.permalink }
-				rel={ showProductLink ? 'nofollow' : '' }
+				rel={ 'nofollow' + ( rel && ' ' + rel ) }
+				target={ linkTarget || undefined }
```

Comment:
Since `linkTarget` is an optional prop, wouldn't it already be `string | undefined`?